### ISSUE TITLE
"Game Version" doesn't seem to updated along with the addon

### ISSUE THREAD

**Issue by linaori** [WowUp/WowUp#473]
**Describe the bug**
After a bug report on Discord that the game version wasn't updating, I tried to reproduce it and seems to never update this. The problem seems to be in the code where it only updates this value when no value is known, this value is always known because we had a previous value, thus never updates it.

```typescript
if (!addon.gameVersion) {
    addon.gameVersion = await this.getLatestGameVersion(unzippedDirectory, unzippedDirectoryNames);
}
```

**To Reproduce**
Have an addon with 90001 as interface version, update it to a version that has 90002, it won't update addons.json with the new version

**Expected behavior**
That the interface version is updated

---

I suspect the fix is to just remove the if check and always try to fetch the version from the toc file, as this is what wow itself uses.

**Comment by jliddev** [comment]
Use the same solution here as #205 ? seems like a good fit.

**Comment by linaori** [comment]
Fixed via #483
